tests/ui/transmutability/enums/should_pad_variants.stderr

```diff
@@ -2,7 +2,10 @@ error[E0277]: `Src` cannot be safely transmuted into `Dst` in the defining scope
   --> $DIR/should_pad_variants.rs:44:36
    |
 LL |     assert::is_transmutable::<Src, Dst, Context>();
-   |                                    ^^^ `Src` cannot be safely transmuted into `Dst` in the defining scope of `should_pad_variants::Context`.
+   |                                    ^^^
+   |                                    |
+   |                                    `Src` cannot be safely transmuted into `Dst` in the defining scope of `should_pad_variants::Context`.
+   |                                    Reason: DstIsTooBig
```

Comment:
I'd suggest:
```
`Src` cannot be safely transmuted into `Dst` in the defining scope of `should_pad_variants::Context` because the size of `Src` is smaller than the size of `Dst` (YYY bytes).
```